node/src/components/Navbar.js

```diff
@@ -1,19 +1,7 @@
 import JSXComponent from 'metal-jsx';
+import * as Auth from '../auth/Auth';
```

Comment:
I think we should have a new line between imports from libraries vs imports from our own components/files.

i.e.
```javascript
import Component from 'metal-jsx';
import {connect} from 'metal-redux';

import * as Auth from '../auth/Auth';
```

Also you should only import what you need into a file.